authentication/oauth/oauth.go

```diff
@@ -96,3 +98,10 @@ type RevokeRefreshTokenRequest struct {
 	// Extra parameters to be merged into the request body. Values set here will override any existing values.
 	ExtraParameters map[string]string `json:"-"`
 }
+
+// IDTokenValidationOptions allows validating optional claims that might not always be in the ID token.
+type IDTokenValidationOptions struct {
+	MaxAge       time.Duration
+	Nonce        string
+	Organization string
```

Comment:
This is the organization ID, right? Should it be renamed to `OrganizationID`?

Reply:
Yeah it is, have renamed to clarify